config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -101,9 +102,14 @@ private static Configuration buildConfiguration() {
             configTypeName = CURRENT_FILE_INSTANCE.getConfig(
                 ConfigurationKeys.FILE_ROOT_CONFIG + ConfigurationKeys.FILE_CONFIG_SPLIT_CHAR
                     + ConfigurationKeys.FILE_ROOT_TYPE);
+
+            if (StringUtils.isBlank(configTypeName)) {
+                throw new NotSupportYetException("can not get config type: " + configTypeName);
```

Comment:
configTypeName is blank，why not print it out explicitly？

Reply:
you are right ,i change the message to "config type can not be null"